service-front/module/Application/src/Services/OpgApiService.php

```diff
@@ -366,6 +366,6 @@ public function getServiceAvailability(): array
             throw new OpgApiException('Service availability data missing!');
         }
 
-        return (new DependencyCheck($this->responseData))->getProcessedStatus();
+        return (new DependencyCheck($this->responseData))->toArray();
```

Comment:
If you return an array, you're throwing away the data structure and losing any benefit of having an object/properties. Better to return `DependencyCheck` itself so you can be confident what properties are available.
```suggestion
        return new DependencyCheck($this->responseData);
```
Or, if you want to use array shapes instead of an object, make DependencyCheck [an invokable](https://www.php.net/manual/en/language.oop5.magic.php#object.invoke) since it doesn't need to carry state.